jax/experimental/jax2tf/examples/tflite/mnist/mnist.py

```diff
@@ -0,0 +1,263 @@
+  interpreter.allocate_tensors()
+  input_tensor_index = interpreter.get_input_details()[0]['index']
+  output = interpreter.tensor(interpreter.get_output_details()[0]['index'])
+
+  # Run predictions on every image in the "test" dataset.
+  prediction_digits = []
+  for test_image in test_ds['image']:
+    # Pre-processing: add batch dimension and convert to float32 to match with
+    # the model's input data format.
+    test_image = np.expand_dims(test_image, axis=0).astype(np.float32)
+    interpreter.set_tensor(input_tensor_index, test_image)
+
+    # Run inference.
+    interpreter.invoke()
+
+    # Post-processing: remove batch dimension and find the digit with highest
+    # probability.
+    digit = np.argmax(output()[0])
+    prediction_digits.append(digit)
+
+  # Compare prediction results with ground truth labels to calculate accuracy.
+  accurate_count = 0
+  for index in range(len(prediction_digits)):
+    if prediction_digits[index] == test_ds['label'][index]:
+      accurate_count += 1
+  accuracy = accurate_count * 1.0 / len(prediction_digits)
+
+  return accuracy
+
+
+def main(_):
+  train_ds, test_ds = get_datasets()
+  optimizer = train(train_ds)
+  def predict(image):
+    return CNN().apply({'params': optimizer.target}, image)
+
+  # Convert Flax model to TF function.
+  tf_predict = tf.function(
+      jax2tf.convert(predict, with_gradient=False, enable_xla=False),
+      input_signature=[
+          tf.TensorSpec(shape=[1, 28, 28, 1], dtype=tf.float32, name='input')
+      ],
+      autograph=False)
+
+  # Convert TF function to TF Lite format.
+  converter = tf.lite.TFLiteConverter.from_concrete_functions(
+      [tf_predict.get_concrete_function()])
+  converter.target_spec.supported_ops = [
+      tf.lite.OpsSet.TFLITE_BUILTINS,  # enable TensorFlow Lite ops.
+      tf.lite.OpsSet.SELECT_TF_OPS  # enable TensorFlow ops.
+  ]
+  tflite_float_model = converter.convert()
+
+  # Show model size in KBs.
+  float_model_size = len(tflite_float_model) / 1024
+  print('Float model size = %dKBs.' % float_model_size)
+
+  # Re-convert the model to TF Lite using quantization.
+  converter.optimizations = [tf.lite.Optimize.DEFAULT]
+  tflite_quantized_model = converter.convert()
+
+  # Show model size in KBs.
+  quantized_model_size = len(tflite_quantized_model) / 1024
+  print('Quantized model size = %dKBs,' % quantized_model_size)
+  print('which is about %d%% of the float model size.'
+        % (quantized_model_size * 100 / float_model_size))
+
+  # Evaluate the TF Lite float model. You'll find that its accurary is identical
+  # to the original Flax model because they are essentially the same model
+  # stored in different format.
+  float_accuracy = evaluate_tflite_model(tflite_float_model, test_ds)
+  print('Float model accuracy = %.4f' % float_accuracy)
+
+  # Evalualte the TF Lite quantized model.
+  # Don't be surprised if you see quantized model accuracy is higher than
```

Comment:
Suggestion:

```suggestion
  # Don't be surprised if the quantized model's accuracy is higher than
```

(for a11y)